docs/modify-tables.md

```diff
@@ -0,0 +1,45 @@
+# Create and modify tables
+
+Creating Tables:
+
+Think of the CREATE TABLE command like building a filing cabinet. It lets you create new folders (tables) within your database to organize information neatly. Each table has rows (like individual files) and columns (like sections within a file) to hold specific details. This command makes storing and retrieving data much more straightforward.
+
+Modifying Tables:
+
+Imagine you need to adjust your filing system. The ALTER TABLE command acts like a toolkit that allows you to change the structure of existing tables. You can add new sections (columns), remove old ones, or change how information is stored [(data types)](data-types-basic.md). This command helps you adapt your database to new needs or improve its efficiency.
+
+Things to Watch Out For:
+
+* Data Loss: Be careful when modifying tables! Deleting a section (column) or changing its format might erase existing data in that section.
+
+* Slowdowns: Altering large tables or making complex changes can slow down the database, especially during busy times. It might take longer for things to work while the changes are applied.
+
+* Locks: MySQL might temporarily lock the tables you're working on when making changes. This operation means other users can't access or modify that data until the changes are complete, which can cause delays for others.
+
+
+## Create Tables
+
+You use the CREATE TABLE command to create a new table in Percona Server for MySQL. This command allows you to specify the table's name and structure, including the columns it contains and their data types.
+
+The following is an example of `CREATE TABLE`:
+
+```{.bash data-prompt="mysql>"}
+mysql> CREATE TABLE users (
+    id INT AUTO_INCREMENT PRIMARY KEY,
+    username VARCHAR(50),
+    email VARCHAR(100)
+);
+```
+
+## Modifying Tables
```

Comment:
## Modifying Tables -> ## Modify tables

Also, change all subsection titles to lower case, for example, Modify Tables -> Modifying tables